src/javascript/expression.rs

```diff
@@ -113,6 +113,18 @@ impl Generator {
     }
 }
 
+pub fn constant_expression<'a, T, Y>(expression: &'a Constant<T, Y>) -> Output<'a> {
```

Comment:
I've put this in the expression file as I suspect it will be able to reuse quite a few things like the string/array render function